web/src/onboarding_steps.ts

```diff
@@ -57,10 +61,31 @@ function narrow_to_dm_with_welcome_bot_new_user(
     }
 }
 
+function show_onboarding_video(): void {
+    if (ONE_TIME_NOTICES_TO_DISPLAY.has("onboarding_video")) {
+        const html_body = render_onboarding_video_modal({
+            video_src: "../../static/videos/onboarding-video.mp4",
```

Comment:
I'm not sure whether this path is available in a production release tarball; we should check that (probably `tar -tf` on something from download.zulip.com is a fine way to check?).

Hmm, I'm also not sure the `../../` pathing is safe; I think it would be better to write just `/static/...` for a relative path? I guess we have `../../static/` paths in existing templates for accessing emoji, so maybe it's fine.

Reply:
> I'm not sure whether this path is available in a production release tarball; we should check that (probably tar -tf on something from download.zulip.com is a fine way to check?).

Verified. It's present.